### DISCUSSION TITLE
Invalid command arguments

### DISCUSSION THREAD

**Discussion by mkundu1** [ansys/pyfluent#1993] [Ideas]
From @h-krishnan:

I notice that an exception is being thrown if a name of a keyword parameter passed to a command doesn't match any argument. Should we rather noisily ignore such parameters? This will allow "forward compatibility" : if an argument is added to a command in a future release, a journal fragment of that release will work with previous releases as well.

**Comment by h-krishnan** [comment]
What if the journal fragment has a line to set a path that doesn't exist in an older version?

Maybe we need this as a configurable option?

**Comment by seanpearsonuk** [comment]
@mkundu1 @h-krishnan I am happy with the current behaviour, as it is consistent with other behaviour and most likely indicates a mistake. But we can move this to discussions so that it is retained in the records.
